frontend/src/app/components/containers/complaints/outcomes/animal-outcomes/add-drug.tsx

```diff
@@ -79,10 +88,12 @@ export const AddDrug: FC<props> = ({
   };
 
   const handleAmountUsed = (input: string) => {
+    if(isPositiveInt(input))
```

Comment:
I don't think this is a good idea for a couple reasons. The big one is that units can be decimals, 1.5, 2.5 etc, and this effectively prevents someone from using decimals. Additionally in the edge case where someone uses a negative value, it would be easier to flip the negative to a positive value

Reply:
Agree about the decimal, for some reason I thought this was only ints. Disagree about flipping the value, think that could be confusing for the user and just preventing the negative input is more intuitive.

Reply:
if you don't want to flip the negative to a positive then you'll need to provide a message to the user that they can't use a negative value otherwise it will be very confusing to the user why they can't add a value there.